src/GraphQLParser/LexerContext.cs

```diff
@@ -90,8 +76,7 @@ public Token ReadNumber()
 
             if (nextCode >= 48 && nextCode <= 57)
```

Comment:
```suggestion
            if (nextCode >= '0' && nextCode <= '9')
```
I suggest that the character representations be used rather than the value throughout the code.

Reply:
done